### DISCUSSION TITLE
Request to Increase TTS Limit for Startup Program

### DISCUSSION THREAD

**Discussion by nithisb** [deepgram/community#924] [General help] [answered]
We are currently participating in the Deepgram Startup Program and utilizing our startup credits. As part of our ongoing development, we have encountered a limitation with the Text-to-Speech (TTS) feature, where we are restricted to only 3 concurrent requests. We would like to request an increase in the limit for concurrent TTS requests to better support our project.

Additionally, could you please guide us on how to directly connect with the Startup Program support team for any future queries or assistance?

We would appreciate your help in increasing our concurrent TTS request limit. Please let us know if any additional information is required.

Thank you for your support!

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by jkroll-deepgram** [comment, marked as answer]
Hi @nithisb, could you please explain more about how the 3 concurrent TTS requests is too low for you, and how many concurrent requests you're looking for?

We set that number because a TTS request takes about 250 ms to complete, and is needed perhaps every 10-20 seconds for a voicebot reply in a 2-party call. So three concurrent requests can support around 120-240 concurrent conversations with a human.

We have a startup program Slack channel - is there an email you can share that we can add to that channel? Besides Slack, GitHub and Discord are also two good places to connect with the Deepgram user [community](https://community.deepgram.com/).

**Reply by nithisb** [reply]
Hi @jkroll-deepgram Thank you for your explanation about the 3 concurrent TTS requests. I understand the logic behind the current limit. However, in our case, we're developing a voicebot solution that handles dynamic user interactions in a high-traffic environment, where requests for TTS are frequent and need to be processed swiftly.

The 250 ms per request completion time sounds reasonable, but given our expected volume of concurrent conversations, we anticipate needing to support more than 120-240 concurrent sessions. I will discuss with our team to determine the exact number of concurrent requests we’d like to request for an increase.

Regarding the Slack channel, please add our team via [dev-team@bookingbee.ai](mailto:dev-team@bookingbee.ai). We'll also check out the GitHub and Discord communities.

Looking forward to your response.
